projects/user_publication/utils.py

```diff
@@ -3,12 +3,37 @@
 import re
 from collections import Counter
 from difflib import SequenceMatcher
+from unidecode import unidecode
```

Comment:
It looks like we're not taking a dependency on this in our poetry config even though it's a third-party library.

Reply:
Good catch, @AnishReddyRavula not sure if that file is just missing from the commit, or you'll need to run `poetry add --lock` to add it https://python-poetry.org/docs/cli/#add.

Reply:
For some reason, I can import this library without making any changes to the poetry.

Reply:
It might be a dependency of something else we use (it wouldn't surprise me if django uses it), but the requirement should be made explicit in case whatever it's included with drops it.